lib/msal-node/src/client/ClientAssertion.ts

```diff
@@ -85,4 +95,16 @@ export class ClientAssertion {
     private isExpired(): boolean {
         return this.expirationTime < TimeUtils.nowSeconds();
     }
+
+    public static parseCertificate(publicCertificate: string): Array<string> {
+        const re = /-----BEGIN CERTIFICATE-----\n(.+?)\n-----END CERTIFICATE-----/gm;
```

Comment:
Can this get a more descriptive variable name?

Reply:
Done